js/task-controller.js

```diff
@@ -0,0 +1,105 @@
+export default class TaskController {
+
+    _tasks = [];
+
+    currentTasks_container = document.querySelector("#currentTasks");
+    completedTasks_container = document.querySelector("#completedTasks");
+
+    setTasks(newTasks) {
```

Comment:
use setter instead

Reply:
now I use getter and setter
```suggestion
    set tasks(newTasks) {
        this._tasks = newTasks;
    }

```